pyproject.toml

```diff
@@ -0,0 +1,103 @@
+[tool.poetry]
+name = "pytket-aqt"
+version = "0.29.0"
+description = "Extension for pytket, providing access to AQT backends"
+authors = ["TKET development team"]
```

Comment:
I think we had set an email address here in the past? Is that gone on purpose?

Reply:
Ooops, yeah I'll add it back